README.md

```diff
@@ -103,68 +95,29 @@ export default AbstractModal.extend({
   willDestroyElement () {
     this.get('modalForms').setModalActive(false)
   },
-
-  formData: {
-    state: {},
-    isValid: false
-  },
-
-  clearForm () {
-    this.set('formData', {
-      state: {},
-      isValid: false
-    })
-  },
-
-  actions: {
-    cancel () {
-      this.clearForm()
-      this.sendAction('dismiss')
-    },
-
-    formValueChanged (formState) {
-      this.set('formData.state', formState)
-    },
-
-    onValidation (info) {
-      this.set('formData.isValid', info.valid)
-    },
-
-    save () {
-      if (this.get('formData.isValid')) {
-        this.sendAction('save', this.get('formData.state'))
-        this.sendAction('dismiss')
-        this.clearForm()
-      }
-    }
-  }
 })
 ```
-##### 2) Parent Component Template with ember-perfectscroll
-###### a) Wrap container element with ember-perfectscroll helper
-###### b) Footer element must have class 'actions'
-###### c) Specify options for [ember-perfectscroll](https://www.npmjs.com/package/ember-perfectscroll) helper such as wheelSpeed, minScrollbarLength, etc.
-
-For more documentation on options: ember-perfectscroll uses [perfect-scrollbar](https://github.com/noraesae/perfect-scrollbar)
-
+#### Component template using ember-block-slots
 Below is an example of template.hbs
 ```handlebars
-{{#frost-modal-input title='Test title' subtitle='Subtitle'}}
-{{!#frost-modal-input titleComponent='myTitle'}}
-
-  {{#perfect-scroll
-    wheelSpeed=2
-    minScrollbarLength=20
-    maxScrollbarLength=150
-  }}
-  {{frost-bunsen-form
+{{#frost-modal-input
+  title='Test title'
+  subtitle='Subtitle'
+  modalName=modalName as |slot|}}
+  {{#block-slot slot 'openButton' as |item|}}
+    {{item.button
+      text='Open long form with scroll'
+      priority='secondary'
+      size='medium'}}
+  {{/block-slot}}
+  {{#block-slot slot 'form'}}
```

Comment:
Let's rename this `content` since we don't interact directly with the form in the component and this makes the concept more reusable as a general modal